boundary_layer/registry/types/operator.py

```diff
@@ -422,8 +422,9 @@ def _build_task_id(self, execution_context):
             return base_name
 
         suffix_mode = execution_context.referrer.item.get('auto_task_id_mode')
-        if not suffix_mode or suffix_mode == 'item_name':
-            return base_name + '-<<item_name>>'
+        name_var = 'batch_name' if execution_context.referrer.item.get('batching', {'enabled': False})['enabled'] else 'item_name'
+        if not suffix_mode or suffix_mode == name_var:
```

Comment:
Hmm, should probably support either `batch_name` _or_ `item_name` when batching is enabled, but only `item_name` if batching is not enabled.  Thoughts?

Reply:
Oh yup, if I had read this comment first I would have type a lot less up there ^^ haha

Reply:
On second thought, I don't think it makes sense to use `item_name` in a batching scenario.  There's not really a good way to construct the item name in that case, so I think we'd _have_ to use `batch_name`.  Let me know if I'm missing something here...  Otherwise, I'll add some validation around this.

Reply:
👍